app/jobs/include_publisher_in_payout_report_job.rb

```diff
@@ -35,15 +35,14 @@ def perform(arguments = {})
     when UPHOLD
       potential_payment_job = Payout::UpholdService
     when BITFLYER
-      potential_payment_job = Payout::BitflyerService
+      return Payout::BitflyerService.build.perform(publisher: publisher,
```

Comment:
IMO I'd rather have all code follow one convention rather than going half-way and implementing a new pattern and any maintainer has to maintain more mental models than necessary. Furthermore, I don't see the benefit of the builder pattern when we want all code to use an inheritance pattern.

Reply:
Discussed offline, we'll refactor this later.